### DISCUSSION TITLE
Indexing one document at a time a bad idea?

### DISCUSSION THREAD

**Discussion by eklem** [fergiemcdowall/search-index#595] [Q&A] [answered]
How is it with term vectors and general health of search-index data if I index one document at a time? Will this make the term vectors not so good?

The reason I ask is that I'm thinking of finishing the browsercrawler, and for that crawler to work it would be ideal to index documents one by one, whenever it's crawled.

**Comment by fergiemcdowall** [comment, marked as answer]
It will be slightly slower than indexing batches, but it other than that it should be fine
